### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by bim240**

### CODE REVIEW

File: array-methods/7-array/index.html [hunk 1]
```diff
@@ -11,36 +11,76 @@
...
-        // your code goes here
+        return got.houses.reduce((sum,v,i) => sum += got.houses[i].people.length , 0).sort();
         console.log(got);
       }
 
       function peopleByHouses() {
-        // your code goes here
+        console.log((got.houses.map( (v, i) => {
+        return (`${got.houses[i].name} : ${got.houses[i].people.length}`)
+        })).sort());
```
Comment (nnnkit) [inline array-methods/7-array/index.html]: Look at the output and do it again.

File: array-methods/7-array/index.html [hunk 2]
```diff
@@ -11,36 +11,76 @@
...
       }
 
       function everyone() {
-        // your code goes here
+        return got.houses.reduce((arr,v, i)=> {
+        (got.houses[i].people.filter((v1, j)=> 
+        arr.push((got.houses[i].people[j].name))));
+    return arr;
+  },[])
```
Comment (nnnkit) [inline array-methods/7-array/index.html]: Do it again.

File: string-methods/01.md [hunk 3]
```diff
@@ -5,7 +5,12 @@ Write a JavaScript function to check whether an `input` is a *string or not*.
 
 Example:
 ```js
-  // your code goes here
+  function is_string (n) {
+    if (typeof(n) == "string")
+      return true;
+    else
+      return false;
+  }
```
Comment (nnnkit) [inline string-methods/01.md]: Naming convention https://github.com/airbnb/javascript#naming-conventions

**Review by nnnkit — APPROVED** [review]
Go through comments and re-factor your code.

File: array-methods/7-array/index.html (hunk 1 above)
Comment (bim240) [reply array-methods/7-array/index.html]: done

File: string-methods/01.md (hunk 3 above)
Comment (bim240) [reply string-methods/01.md]: Done

File: array-methods/7-array/index.html (hunk 2 above)
Comment (bim240) [reply array-methods/7-array/index.html]: done

**Pull request merged**